examples/minimal-mdns/advertiser.cpp

```diff
@@ -214,7 +214,7 @@ int main(int argc, char ** args)
 
     if (gOptions.advertisingMode == AdvertisingMode::kCommisioning)
     {
-        err = chip::Mdns::ServiceAdvertiser::Instance().Advertise(chip::Mdns::CommisioningAdvertisingParameters()
+        err = chip::Mdns::ServiceAdvertiser::Instance().Advertise(chip::Mdns::CommissionAdvertisingParameters()
```

Comment:
maybe we should explicitly set the mode of commisioning here rather than relying on the default.